test/gtest/ucp/test_ucp_am.cc

```diff
@@ -578,6 +578,11 @@ UCS_TEST_P(test_ucp_am_nbx_dts, long_zcopy_send, "ZCOPY_THRESH=1",
     test_am(64 * UCS_KBYTE);
 }
 
+UCS_TEST_P(test_ucp_am_nbx_dts, send_eager_flag, "RNDV_THRESH=128")
+{
+    test_am(64 * UCS_KBYTE, UCP_AM_SEND_FLAG_EAGER);
```

Comment:
can we somehow test it actually used rndv?

Reply:
it uses eager and it does this check in `am_data_handler` on line 426
`EXPECT_FALSE(rx_param->recv_attr & UCP_AM_RECV_ATTR_FLAG_RNDV);`

Reply:
how about expecting flag RNDV in the FLAG_RNDV case?

Reply:
it is done on line 603